test/steward_admin_change.js

```diff
@@ -4,15 +4,15 @@ const {
   ether,
   expectEvent,
   balance,
-  time
+  time,
```

Comment:
These trailing commas caused some merge conflicts, because it changed code all over the place, but no stress.